qa/tasks/cephfs/test_admin.py

```diff
@@ -2011,3 +2026,153 @@ def test_fs_authorize(self):
                 args=(f'fs authorize {self.fs.name} {self.CLIENT_NAME} / '
                       f'{wrong_perm}'), retval=self.EXPECTED_ERRNO,
                 errmsgs=self.EXPECTED_ERRMSG)
+
+
+class TestFSFail(TestAdminCommands):
+
+    CLIENTS_REQUIRED = 1
+
+    def test_with_health_warn_oversize_cache(self):
+        health_warn = 'MDS_CACHE_OVERSIZED'
+        self.run_ceph_cmd('config set mds mds_cache_memory_limit 10M')
+        self.run_ceph_cmd('config set mds mds_health_cache_threshold 1.00000')
+        tar_link = 'https://download.ceph.com/qa/linux-6.5.11.tar.xz'
+        tar_name = 'linux-6.5.11.tar.xz'
+        #self.mount_a.run_shell(f'wget {tar_link} ')
+        self.mount_a.run_shell(f'cp ~/Downloads/{tar_name} ./')
```

Comment:
@rishabh-d-dave Have you checked TestClientLimits._test_client_pin which simulates a MDS_CACHE_OVERSIZED warning without untarring the linux kernel.